src/lio/os/file.c

```diff
@@ -1128,7 +1128,7 @@ int va_timestamp_set_attr(lio_os_virtual_attr_t *va, lio_object_service_fn_t *os
     n = (int)(long)va->priv;  //** HACKERY ** to get the attribute prefix length
     key = &(fullkey[n+1]);
 
-    if (strlen(fullkey) < n) {  //** Nothing to do so return;
+    if ((int)strlen(fullkey) < n) {  //** Nothing to do so return;
```

Comment:
Well, the name is probably the smallest thing. Tiem for the ole thesaurus!

To be clear, I also agree that unsigned variables are the worst. Unfortunately the comparison rules between mixed-signedness values is even worser, so we're kinda stuck. Especially since a lot of the static analysis/sanitization stuff will trip all over it.

Anyway, time to find dinner....